src/components/token/components/section_3.jsx

```diff
@@ -27,31 +43,34 @@ const SectionThree = () => {
     ],
     []
   )
+
+
+  const {
+    data: holdersData = {
+      bsc: '1,642',
```

Comment:
@ahmednabil310 I think you can remove the hardcoded numbers

Reply:
done, btw they were a backup default values